.github/actions/install_python_dependencies/action.yml

```diff
@@ -0,0 +1,15 @@
+name: Install Kedro and other Python Dependencies
```

Comment:
[Nit] For our project it make sense to have kedro installed here. But in a general case as the file name says - we should only install python dependencies and have kedro installation in a separate step. But I am okay if the team feels otherwise.

Reply:
@ravi-kumar-pilla 
If you notice workflows where this action is used in its step name, it says ‘Install Kedro and other Python Dependencies.’ I want to avoid repeating the step of ‘Install Kedro’. But I can rename the action file to ‘install_kedro_and_python_dependencies’ for more clarity.

Reply:
`install_kedro_and_python_dependencies` this is more clear but also a bit lengthy. I would suggest to have `install_kedro_and_python_dependencies` for better readability